pkgs/applications/audio/rosegarden/default.nix

```diff
@@ -0,0 +1,39 @@
+{ stdenv, fetchurl, qt4, pkgconfig, ladspaPlugins, ladspaH,
+  dssi, liblo, liblrdf, fftwSinglePrec, libsndfile,
+  libsamplerate, perl, makedepend, libjack2,
+  withLirc ? false, lirc ? null } :
+
+let rosegarden = rec {
```

Comment:
Can you please put everything into the derivation instead of using this let? It's pointless.